### ISSUE TITLE
google-maps: plugin can not create map under IOS version > 17.2

### ISSUE THREAD

**Issue by ngmiduc** [ionic-team/capacitor-plugins#2110]
### Plugin(s)
```
"@capacitor/google-maps": "^5.4.1"
```

### Capacitor Version
```
Latest Dependencies:

  @capacitor/cli: 6.0.0
  @capacitor/core: 6.0.0
  @capacitor/android: 6.0.0
  @capacitor/ios: 6.0.0

Installed Dependencies:

  @capacitor/cli: 5.0.4
  @capacitor/android: 5.0.4
  @capacitor/core: 5.7.0
  @capacitor/ios: 5.0.4
```

### Platform(s)
IOS

### Current Behavior
When the app is freshly installed on IOS 17.5, than the app crashes when the google map plugins create the map. After that the app can be sometimes opened again, after the app crashed. To reproduce this, delete the app again and install it again with xcode. 

### Expected Behavior
App should not crash after the app is lunched for the first time.

### Code Reproduction
https://stackblitz.com/edit/angular-yz24zb-dbfvbk?file=src%2Fapp%2Fexample.component.ts

1. add  API KEY to componentn:  `const apiKey = 'xxx';`
2. build app to IOS
3. run on emulator in IOS = 17.5
4. start up

### Other Technical Details
Reproduction code can be downloaded from stackblitz

**Comment by focafull** [comment]
I think we've got the same problem, but could never really reproduce it in production.

**Comment by ngmiduc** [comment]
> I think we've got the same problem, but could never really reproduce it in production.

Yeah, we have the same issue maybe. It's not easy to reproduce. We have some users that experiences app crashes. I am not sure if it is related to this issue but it is the only thing that we found out during debugging.

**Comment by ngmiduc** [comment]
Ah, we found a relevant issue here: https://github.com/googlemaps/ios-maps-sdk/issues/20

**Comment by ngmiduc** [comment]
Ok. Downgrading xcode to 15.2 didn't resolve the issue. My emulator is still crashing.

I tried to install the new version of the GoogleMaps pod 8.4.0. This required setting up the min deployment target to IOS 14 and I needed to build Google-Maps-iOS-Utils with rosetta on M1, since Google-Maps-iOS-Utils latest version doesn't work with the latest google maps version. I left it with an older version. 

After that, I could run again on an IOS 17.5 device in the simulator-rosetta.

**Comment by ngmiduc** [comment]
Ok. After some more debugging, I found out that if I build the app with Rosetta, than it app works again after a fresh installation. I am now not sure anymore what the issue is.... - would need some help from here on now...
